docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -1,75 +1,71 @@
 ////
 This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
-https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
 = Building Quarkus apps with Quarkus Command Line Interface (CLI)
+:extension-status: experimental
```

Comment:
Is the status-include there? Because if not, this is useless. Maybe check the rendering?

Reply:
isn't that what include::./attributes.adoc[]
 does ? but yes will check the rendering again

Reply:
No, you have to add `include::./status-include.adoc[]` yourself. `attributes.adoc` just contains the attributes.

Reply:
fixed and tested locally that the preview message does show up.